### DISCUSSION TITLE
Se mueve los elemento al hacer zoom

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#365] [Q&A]
Compañeros no se si puedan ayudarme, estoy haciendo la practica de CSS pero al hacer zoom o alejar en mi practica todos los elementos se mueven como pueden ver en la imagen, esto como podría solucionarse?

![image](https://user-images.githubusercontent.com/88987587/158034061-1455d02f-d1a0-44d4-84e7-b884e6dc139e.png)

**Comment by hackit92** [comment]
Eso es debido a que la página busca adaptarse y ser responsiva a esos cambios (muy útil para poder ver la página en múltiples resoluciones y dispositivos), sin embargo al no contar con el código adecuado pasa eso. En esta práctica no estamos buscando que sea responsiva aún, por lo que te recomiendo centrarte en que se vea bien en la resolución principal (escritorio)

**Comment by Ingrzs** [comment]
Eso se  hace manejando los elementos con porcentaje y meterle responsive para quede perfectamente a la medida segun el dispositivo

**Comment by doguedogue** [comment]
Con el viewport (meta del head) controlas que hagan o no zoom
